t/01-test-utilities.t

```diff
@@ -35,16 +35,18 @@ subtest 'warnings in sub processes are fatal test failures' => sub {
             isnt(shift, 0, 'exit code of test failure is non-zero');
             $test_would_have_failed = 1;
         });
-    combined_like {
+    my $out = combined_from {
         # start a sub process like the test helper do and simulate a Perl warning
         OpenQA::Test::Utils::_setup_sigchld_handler 'test-process-1', start sub {
+            sleep 1;                                                     # Give Utils.pm a chance to install $SIG{CHLD}
```

Comment:
We now need an "uncoverable statement" in the comment as well

Reply:
ah right. done